scalableminds/webknossos
# Fix skeleton search

The skeleton search did not expand the group of the skeleton it selected. I changed the code so that all parent groups of a selected skeleton via the search are not automatically expanded.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open https://fixskeletontabsearch.webknossos.xyz/annotations/6710b779010000af00c4b903#3584,3584,1024,0,2
- open the skeleton tab
- collapse all groups in case they aren't
- use the skeleton search with the term `search me` and go through all selection
- all necessary skeleton groups should auto-expand in case the tree selected via the search is within a collapsed group
- The selected trees should be highlighted as well

### Issues:
- fixes #8057

------
(Please delete unneeded items, merge only when none are left open)
